packages/suite/src/support/messages.ts

```diff
@@ -7924,4 +7924,16 @@ export default defineMessages({
         id: 'TR_DISABLE_AUTOPAUSE_COINJOIN',
         defaultMessage: "Don't pause",
     },
+    TR_AMOUNT_SENT: {
+        id: 'TR_AMOUNT_SENT',
+        defaultMessage: 'Amount sent',
+    },
+    TR_AMOUNT_SPENT: {
+        id: 'TR_AMOUNT_SPENT',
+        defaultMessage: 'Amount spent',
```

Comment:
https://github.com/trezor/trezor-suite/pull/7989#issuecomment-1495889804
```suggestion
        defaultMessage: 'Total amount',
```
?

Reply:
Probably to align with the copy in FW. https://github.com/trezor/trezor-suite/pull/7989#issuecomment-1495876272

Reply:
7b165ac6d